lib/client/node.go

```diff
@@ -180,4 +193,32 @@ func (h *nodes) convertKVPairToAPI(d *model.KVPair) (unversioned.Resource, error
 	}
 
 	return apiNode, nil
-}
\ No newline at end of file
+}
+
+// initGlobalDefaults initialises any global default value that is not yet
+// initialised, this processing will not overwrite keys that are already
+// present in the datastore.
+func (h *nodes) initGlobalDefaults() error {
+	// Make sure the Ready flag is initialised in the datastore
```

Comment:
Ugh this is the problem with moving Country ... I can't make up my mind anymore ;-)